### ISSUE TITLE
Cirular import of ModuleLint

### ISSUE THREAD

**Issue by grst** [nf-core/tools#1270]
## Description of the bug

I cannot run individual unittests from vscode due to a circular import. 

## Steps to reproduce

Steps to reproduce the behaviour:

```python
import nf_core.modules.lint
```
```pytb
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-90faefaf9286> in <module>
----> 1 import nf_core.modules.lint

~/projects/2020/nf-core-tools/nf_core/modules/__init__.py in <module>
      2 from .create import ModuleCreate
      3 from .test_yml_builder import ModulesTestYmlBuilder
----> 4 from .lint import ModuleLint
      5 from .bump_versions import ModuleVersionBumper
      6 from .module_utils import ModuleException

~/projects/2020/nf-core-tools/nf_core/modules/lint/__init__.py in <module>
     24 import rich
     25 from nf_core.utils import rich_force_colors
---> 26 from nf_core.lint.pipeline_todos import pipeline_todos
     27 import sys
     28 

~/projects/2020/nf-core-tools/nf_core/lint/__init__.py in <module>
     22 import nf_core.lint_utils
     23 from nf_core.lint_utils import console
---> 24 from nf_core.modules import ModuleLint
     25 
     26 log = logging.getLogger(__name__)

ImportError: cannot import name 'ModuleLint' from partially initialized module 'nf_core.modules' (most likely due to a circular import) (/home/sturm/projects/2020/nf-core-tools/nf_core/modules/__init__.py)
```

## System

- Hardware: desktop
- Executor: local
- OS: Arch linux
- Version of nf-core/tools: checked out at `8635612b3ee3b1e414414553c4fd72cb048464d3`
- Python version: 3.8.10

## Nextflow Installation

irrelevant

**Comment by grst** [comment]
Sorry, I hit the submit button too early. The issue is now complete.

**Comment by ewels** [comment]
Should be fixed by #1314 - please reopen and make some noise if that's not the case 😅
